packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -35,7 +35,8 @@ TODO: Remove this section if there are not any general updates.
 
 ## Debugger updates
 
-TODO: Remove this section if there are not any general updates.
+* The console in the debugger now soft wraps lines.
```

Comment:
Verbiage nit: entries should be phrased in the past tense (e.g. "Added XYZ" instead of "Add XYZ").

Reply:
Done.